src/main/java/pl/byczazagroda/trackexpensesappbackend/model/User.java

```diff
@@ -56,11 +56,11 @@ public class User implements Serializable {
 
     @NotBlank
     @Size(min = 6, max = 120)
-    @Pattern(regexp = "^[\\w-.]+@([\\w-]+.)+[\\w-]{2,4}$")
+    @Value("${regex.pattern.email}")
     private String email;
 
     @NotBlank
-    @Size(min = 8, max = 100)
+    @Value("${regex.pattern.password}")
```

Comment:
Is regex needed here?

Reply:
It is not necessary by the fact that the same regex is in DTOs but I go on the assumption that it is better to add. If I have to delete please tell me.

Reply:
Yes, you have right.  I removed validation with pattern in User class.